### DISCUSSION TITLE
I didn't get the Arctic code voult badge

### DISCUSSION THREAD

**Discussion by lazy-geek** [community/community#23430] [New to GitHub]
<p>I recently Contributed to a repo matching criteria for arctic code voult but i didn’t get the badge</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>You would have had to have contributed before the snapshot was taken in February 2020 to be included in the <a href="https://archiveprogram.github.com/">GitHub Arctic Code Vault</a>.</p>
<p>Keep contributing - you never know what you’ll contribute to in the future!</p>

**Comment by lazy-geek** [comment]
<p>Is there going to be another program like arctic code voult again in the future?</p>

**Comment by canuckjacq** [comment]
<p>Very possibly! We did recently have a badge for folks <a href="https://github.blog/changelog/2021-04-19-new-badge-for-developers-who-contributed-to-the-mars-2020-helicopter-mission/">who contributed to software that went to Mars</a>.</p>
